datasets/americas_nli/americas_nli.py

```diff
@@ -0,0 +1,178 @@
+"""
+
+VERSION = datasets.Version("1.0.0", "")
+_DEV_DATA_URL = "https://raw.githubusercontent.com/nala-cub/AmericasNLI/main/dev.tsv"
+_TEST_DATA_URL = "https://raw.githubusercontent.com/nala-cub/AmericasNLI/main/test.tsv"
+
+_LANGUAGES = ("aym", "bzd", "cni", "gn", "hch", "nah", "oto", "quy", "shp", "tar")
+
+
+class AmericasNLIConfig(datasets.BuilderConfig):
+    """BuilderConfig for AmericasNLI."""
+
+    def __init__(self, language: str, languages=None, **kwargs):
+        """BuilderConfig for AmericasNLI.
+
+        Args:
+        language: One of aym, bzd, cni, gn, hch, nah, oto, quy, shp, tar or all_languages
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(AmericasNLIConfig, self).__init__(**kwargs)
+        self.language = language
+        if language != "all_languages":
+            self.languages = [language]
+        else:
+            self.languages = languages if languages is not None else _LANGUAGES
+
+
+class AmericasNLI(datasets.GeneratorBasedBuilder):
+    """TODO"""
+
+    VERSION = VERSION
+    BUILDER_CONFIG_CLASS = AmericasNLIConfig
+    BUILDER_CONFIGS = [
+        AmericasNLIConfig(
+            name=lang,
+            language=lang,
+            version=VERSION,
+            description=f"Plain text import of AmericasNLI for the {lang} language",
+        )
+        for lang in _LANGUAGES
+    ] + [
+        AmericasNLIConfig(
+            name="all_languages",
+            language="all_languages",
+            version=VERSION,
+            description="Plain text import of AmericasNLI for all languages",
+        )
+    ]
+
+    def _info(self):
+        if self.config.language == "all_languages":
+            features = datasets.Features(
+                {
+                    "language": datasets.Value("string"),
+                    "premise": datasets.Value("string"),
+                    "hypothesis": datasets.Value("string"),
+                    "label": datasets.ClassLabel(names=["entailment", "neutral", "contradiction"]),
+                }
+            )
+        else:
+            features = datasets.Features(
+                {
+                    "premise": datasets.Value("string"),
+                    "hypothesis": datasets.Value("string"),
+                    "label": datasets.ClassLabel(names=["entailment", "neutral", "contradiction"]),
+                }
+            )
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            # No default supervised_keys (as we have to pass both premise
+            # and hypothesis as input).
+            supervised_keys=None,
+            homepage="TODO",
```

Comment:
I think we can use this URL here
```suggestion
            homepage="https://github.com/nala-cub/AmericasNLI",
```